pkg/services/sqlstore/stats_test.go

```diff
@@ -108,5 +119,12 @@ func populateDB(t *testing.T) {
 		Role:   models.ROLE_ADMIN,
 	}
 	err = AddOrgUser(cmd)
-	assert.Nil(t, err)
+	assert.NoError(t, err)
```

Comment:
It's better to use require here, since we don't want to continue the test if the check fails (`assert.NoError` will continue):
```suggestion
	require.NoError(t, err)
```